bom/runtime/pom.xml

```diff
@@ -2446,7 +2454,38 @@
                 <artifactId>flyway-core</artifactId>
                 <version>${flyway.version}</version>
             </dependency>
-
+            <dependency>
+                <groupId>org.liquibase</groupId>
+                <artifactId>liquibase-core</artifactId>
+                <version>${liquibase.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>ch.qos.logback</groupId>
+                        <artifactId>logback-classic</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>javax.xml.bind</groupId>
+                        <artifactId>jaxb-api</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.yaml</groupId>
+                <artifactId>snakeyaml</artifactId>
+                <version>${snakeyaml.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.osgi</groupId>
+                <artifactId>org.osgi.core</artifactId>
+                <version>${osgi.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
```

Comment:
Does it really work? We had some bad surprises with star excludes.

Reply:
I would like to avoid the osgi dependency in general. The osgi dependency is used only because of this static code which is not used in the quarkus: https://github.com/liquibase/liquibase/blob/9908038ad4fd6341470feb4d1aa1c5d084d0eaa9/liquibase-core/src/main/java/liquibase/servicelocator/ServiceLocator.java#L38. Is there any way to avoid the dependency?

I will remove the excludes.


[INFO] +- io.quarkus:quarkus-liquibase:jar:999-SNAPSHOT:compile
[INFO] |  +- org.liquibase:liquibase-core:jar:3.8.4:compile
[INFO] |  +- org.yaml:snakeyaml:jar:1.25:compile
[INFO] |  +- io.quarkus:quarkus-jaxb:jar:999-SNAPSHOT:compile
[INFO] |  |  +- org.glassfish.jaxb:jaxb-runtime:jar:2.3.3-b01:compile
[INFO] |  |  |  +- org.glassfish.jaxb:txw2:jar:2.3.3-b01:compile
[INFO] |  |  |  +- com.sun.istack:istack-commons-runtime:jar:3.0.10:compile
[INFO] |  |  |  \- com.sun.activation:jakarta.activation:jar:1.2.1:runtime
[INFO] |  |  \- org.jboss.spec.javax.xml.bind:jboss-jaxb-api_2.3_spec:jar:2.0.0.Final:compile
[INFO] |  +- io.quarkus:quarkus-narayana-jta:jar:999-SNAPSHOT:compile
[INFO] |  |  +- io.smallrye.reactive:smallrye-reactive-streams-operators:jar:1.0.10:compile
[INFO] |  |  |  +- org.eclipse.microprofile.reactive-streams-operators:microprofile-reactive-streams-operators-api:jar:1.0:compile
[INFO] |  |  |  |  \- org.reactivestreams:reactive-streams:jar:1.0.3:compile
[INFO] |  |  |  +- org.eclipse.microprofile.reactive-streams-operators:microprofile-reactive-streams-operators-core:jar:1.0:compile
[INFO] |  |  |  \- io.reactivex.rxjava2:rxjava:jar:2.2.15:compile
[INFO] |  |  +- io.smallrye:smallrye-context-propagation-jta:jar:1.0.11:compile
[INFO] |  |  \- io.smallrye.reactive:smallrye-reactive-converter-api:jar:1.0.10:compile
[INFO] |  \- org.osgi:org.osgi.core:jar:6.0.0:compile